client/src/components/NavBar/NotificationButton/NotificationButton.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState, MouseEvent } from 'react';
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import useStyles from './useSyles';
+import { Badge, Box } from '@material-ui/core';
+import NotificationRequest from '../NotificationRequest/NotificationRequest';
+import { useNotification } from '../../../context/useNotificationContext';
+import { setUnreadNotificationToRead } from '../../../helpers/APICalls/notifications';
+const testNotifi = [
+  {
+    id: '123',
+    type: 'message',
+    isRead: false,
+    sender: '12265eu56eutr',
+    title: '',
+    message: '',
+    created_at: '2/3/2020',
```

Comment:
```suggestion
    createdAt: '2/3/2020',
```

Please use camelCase, ditto throughout!